doc/quickstart/interfaces/torch.rst

```diff
@@ -0,0 +1,183 @@
+.. _torch_interf:
+
+PyTorch interface
+-----------------
+
+
+In order to use PennyLane in combination with PyTorch code, we have to generate "PyTorch-compatible"
+quantum nodes. A basic :class:`QNode` can be translated into a quantum node that interfaces
```

Comment:
we can just say `PyTorch-compatible quantum nodes`. We did not use the quotes on the tensorflow page.